functions/__tests__/endpoints/debug/index.test.js

```diff
@@ -1,7 +1,7 @@
 const testConfig = require("firebase-functions-test")();
 const supertest = require("supertest");
 const { addDocument, queryDocument } = require("../../../utils/database");
-const { app } = require("../../../controllers/expressExample/index");
+const { app } = require("../../../controllers/endpoints/index");
```

Comment:
Let's delete the debug endpoint entirely with the unit test file since we don't want this in production and we now have some structure in our backend.